pkg/kubeapiserver/options/storage_versions.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package options
 
 import (
+	"sort"
```

Comment:
@deads2k didn't realize i also changed here. but it still makes some sense to me.😅 
should we keep these changes so that import declaration could be in the form like:

```
import (
  // go root pkgs

  // github/other pkgs

  // k8s.io pkgs
)
```

or should we trim the change to follow the defacto of `minimun change`